clang/include/clang/AST/Type.h

```diff
@@ -1507,6 +1508,9 @@ class ExtQuals : public ExtQualsTypeCommonBase, public llvm::FoldingSetNode {
       : ExtQualsTypeCommonBase(baseType,
                                canon.isNull() ? QualType(this_(), 0) : canon),
         Quals(quals) {
+    static_assert(alignof(decltype(*this)) % TypeAlignment == 0,
```

Comment:
This feels unnecessary because we've added `alignas(TypeAlignment)` to the class -- it's basically statically asserting that the compiler honored the alignment we requested (which it has to do anyway or else the program is ill-formed), WDYT?

Reply:
Makes sense. I removed this static assert.